src/System.Net.HttpListener/src/System/Net/Managed/HttpResponseStream.Managed.cs

```diff
@@ -150,6 +174,22 @@ internal void InternalWrite(byte[] buffer, int offset, int count)
             }
         }
 
+        internal async Task InternalWriteAsync(byte[] buffer, int offset, int count)
+        {
+            if (_ignore_errors)
+            {
+                try
+                {
+                    await _stream.WriteAsync(buffer, offset, count);
+                }
+                catch { }
+            }
+            else
+            {
+                await _stream.WriteAsync(buffer, offset, count);
```

Comment:
From a performance standpoint, it'd be better to do something like:
```C#
internal Task InternalWriteAsync(byte[] buffer, int offset, int count) =>_ignore_errors ?
    InternalWriteIgnoreErrorsAsync(buffer, offset, count) :
    _stream.WriteAsync(buffer, offset, count);

private async Task InternalWriteAsyncIgnoreErrors(byte[] buffer, int offset, int count)
{
    try { await _stream.WriteAsync(buffer, offset, count).ConfigureAwait(false); }
    catch { }
}
```
That said, I'm refraining from commenting on most perf-related issues, as there look to be a bunch of areas where we could make improvements separately, and I know that at the moment improving the perf of this component isn't a top priority.